src/App.css

```
@@ -1,3 +1,407 @@
+body {
+  font: 14px 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  line-height: 1.4em;
+  background: #f5f5f5;
+  color: #4d4d4d;
+  min-width: 230px;
+  max-width: 550px;
+  margin: 0 auto;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 300;
+}
+
 .App {
+  background: #fff;
+  margin: 130px 0 40px 0;
+  position: relative;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.2), 0 25px 50px 0 rgba(0, 0, 0, 0.1);
+}
+
+h1 {
+  position: absolute;
+  top: -155px;
+  width: 100%;
+  font-size: 100px;
+  font-weight: 100;
   text-align: center;
+  color: rgba(175, 47, 47, 0.15);
+  -webkit-text-rendering: optimizeLegibility;
+  -moz-text-rendering: optimizeLegibility;
+  text-rendering: optimizeLegibility;
+}
+
+.arrow-btn {
+    width: 60px;
+    height: 34px;
+    font-size: 24px;
+    position: absolute;
+    top: 14px;
+    left: -7px;
+    background: none;
+    border: none;
+    outline: none;
+    color: #e6e6e6;
+    transition: 0.5s;
+}
+
+.arrow-btn:hover {
+    color: #d1d1d1;
+    transition: 0.5s;
```

```suggestion
    transition: transform 0.5s;
```
Better to explicitly add `transition-property`. Otherwise, you can face unexpected issues when you decide to change other properties and they start animation.